### ISSUE TITLE
[Feature Request]: "Color Dialog" is in the Main Forms elements - Where is Kryptons' Standard themed equivalent 

### ISSUE THREAD

**Issue by Smurf-IV** [Krypton-Suite/Standard-Toolkit#265]
Moving in from #243 

https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.commondialog?view=net-5.0

**Comment by Smurf-IV** [comment]
What I'm working with:
Default:
![image](https://user-images.githubusercontent.com/2418812/129446978-470a66a0-e947-4b9e-a884-3198ffd2f830.png)

Or after pressing the Custom Colours >> or setting Full Open:
![image](https://user-images.githubusercontent.com/2418812/129447003-102f32df-7838-4e76-a760-73d2ae6be093.png)


Challenge .. Challenges..

**Comment by Smurf-IV** [comment]
oh, ok.. The default code did a good job..
![image](https://user-images.githubusercontent.com/2418812/129447056-c830e17b-9801-47d5-a595-75375be272f2.png)

BUT...
The "Define Custom Colours" does not work as expected - No surprise there.

**Comment by Smurf-IV** [comment]
Hmm.. A new thing.. 
- Need to have functionality to enable / disable elements after a click event !

**Comment by Smurf-IV** [comment]
Done.
![image](https://user-images.githubusercontent.com/2418812/129455155-5579954a-e3ec-4291-82e8-c6763f59a89a.png)

**Comment by sdreb3421** [comment]
I just upgraded my production code to the latest Nightly and noticed that the KryptonColorButton color dialog was updated by this Feature Request. I'm having a font size issue I'm assuming because my palette uses a larger default form font (see image). I'm also seeing the Form Text cutoff, probably because I've adjusted the header content padding in the palette to fit nicely with my form icon. 

Is there a way to specify the Styles/Fonts of the color dialog, or not have it use Krypton styling? And add a default icon to the color dialog?

![image](https://user-images.githubusercontent.com/1911567/134630192-b9a2c47e-9b6d-4af2-8a27-8b1582b59d61.png)
